src/scripts/post_process_externally_managed_content.py

```diff
@@ -0,0 +1,112 @@
+import click
+import os
+import pandas as pd
+import logging
+
+logger = logging.getLogger(__name__)
+
```

Comment:
`nit` / question on Python code style: Just highlighting this line, as typically between functions we have 2 line breaks.

I haven't been using `black` for `mondo-ingest`, just following PyCharm's defaults. I honestly don't mind right now if you / we don't follow any single style standard for `mondo-ingest` until we have more time. But just curious if you are planning to use `black` or anything else for your work in `mondo-ingest`.

Reply:
I would like to but not without a automated linting system. I made this one change now, but anything else - only after mondolib is in existence.